File: 5-kyu/simple-pig-latin/javascript/solution.js
function pigIt(str){
    var l = [];
    str = str.split(" ");console.log(str);
    var i;
    for (i=0; i<str.length; i++) {
        if (!(/[^a-zA-Z0-9]/.test(str[i])) ){ 
            var pig = str[i].slice(1) + str[i][0] + "ay";console.log(pig);
            l.push(pig);
        }  
        else {
            l.push(str[i]);
        }
    }
    return l.join(" ");
}

File: 5-kyu/simple-pig-latin/python/solution.py
def pig_it(text):
    l = []
    for word in text.split():
        if word.isalpha():
            o = word[1:]+word[0] +  "ay"
            l.append(o)
        else:
            l.append(word)    
    return " ".join(l)  